### ISSUE TITLE
Sync

### ISSUE THREAD

**Issue by m-laclair** [warmdev/SublimeOutline#3]
Thanks so much for adding SublimeOutline!  It was the one thing I was missing from Sublime.

One request:  it would be great to have a way to sync from the current file to that location in the Outline.  Some of our cpp modules have more than one class and lot of methods.  When I go to a class near the bottom of the file, I have to scroll the Outline view down and visually search for it to see the class and its methods.  For me, it would best if the Outline would automatically sync when I moved to a new class/method, etc in my file.   If that's not easy/feasible, it would still be very useful if there were a key combo to set the Outline to the location in the file where I'm working.

Thanks!

**Comment by warmdev** [comment]
You can try the following fix using `PackageResourceViewer` or edit the file directly (use "Preference > Browse Packages" to find the file in the "Outline" folder). 

In `outline.py` file, remove line 44 (the line with `return`), and add the following lines (same level of indent as original line 44)

```python
			if u'𝌆' in view.name():
				return
			if view.window() is None:
				return
			# 0. get the current cursor location
			point = view.sel()[0].begin()
			# 1. get the current symbol and its line
			symlist = view.get_symbols()
			symlist.reverse()
			sym_count = len(symlist) - 1
			sym_line = len(symlist) - 1
			for symbol in symlist:
				rng, sym = symbol
				sym_line_begin = view.line(rng).a
				if (point >= sym_line_begin or sym_count == 0):
					sym_line = sym_count
					break
				sym_count = sym_count - 1
			# 3. get the symbol point in symview
			if (view is not None):
				sym_view, sym_group, fb_view, fb_group = get_sidebar_views_groups(view)
			if (sym_view is not None):
				sym_point_start = sym_view.text_point(sym_line, 0)
				# 4. center symview at the point
				sym_view.show_at_center(sym_point_start)
				sym_view.sel().clear()
				sym_view.sel().add(sym_view.line(sym_point_start))
				view.window().focus_view(view)
```

**Comment by warmdev** [comment]
Implemented in release [0.1.3 ](https://github.com/warmdev/SublimeOutline/releases/tag/0.1.3)
